scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/matcher/simplify/Identity.java

```diff
@@ -35,9 +35,10 @@
 import org.scijava.ops.engine.BaseOpHints.Simplification;
 import org.scijava.ops.spi.Op;
 import org.scijava.ops.spi.OpClass;
+import org.scijava.priority.Priority;
 
 @OpHints(hints = { Simplification.FORBIDDEN })
-@OpClass(names = "simplify, focus")
+@OpClass(names = "simplify, focus, identity", priority = Priority.LAST)
```

Comment:
Why does this op need to be an `OpClass` rather than a more concise `OpField` using a lambda?

Reply:
I mean, you'll still have the `OpCollection` boilerplate, but just for you I made it into a `OpField`!